06_iito/index.tex

```diff
@@ -1 +1,34 @@
-\chapter{Inter Island Trade Organisation (IITO)}
\ No newline at end of file
+\chapter{Inter Island Trade Organisation (IITO)}
+
+The role of IITO is to facilitate inter-island communication and the giving/receiving of gifts. In a perfect simulation unrestricted communication between islands would be ideal but this would ultimately result in the simulation being unimplementeable due to the islande becoming too complex. Instead, specific forms of communication were chosen to sensibly restrict the island complexity while still allowing for interesting simulations.
+
+\section{Inter-Island Communication}  
+\label{sec:IITO:inter_island_communication}
+
+Islands being able to communicate independently from the main governing body allows for more complex island behaviour. With the designed system, islands can:
+
+\begin{itemize}
+    \item Form a group with other islands for collective foraging.
+    \item Decide where their foraging group will forage.
+    \item Inform other islands of the amount they intend to to donate to the common pool.
+    \item Share voting history with other islands.
+    \item Share tax amount history with other islands.
+    \item Share the amount of resources they have with other islands.
+\end{itemize}
+
+\textbf{Team Foraging: }Allowing islands to decide between each other on the groups that they will forage in and where that group will forage means that they can work with islands that they trust while avoiding islands that have misbehaved or broken rules in the past. Islands may also want to avoid foraging with other islands that have contributed less than expected in the past if they suspect that they may have been deliberately freeloading.
```

Comment:
suggested re-writing:

It allows islands to decide between each other whether they will forage together and where they will forage. In this way, islands can cooperate with the others that they trust while avoiding other islands that they deem to have misbehaved or broken rules in the past.  If islands suspect that some specific islands have deliberately freeloaded in the previous forages, they may also want to avoid foraging with such islands that have contributed less than what they were supposed to.